src/styles/mixins/utils.scss

```diff
@@ -46,3 +46,30 @@
   width: 1em;
   height: 1em;
 }
+
+@mixin font-base {
+  line-height: var.$line-height;
+  font-weight: normal;
+  color: var.$color-text;
+}
+
+@mixin font--tiny {
```

Comment:
we are using xsmall for the icons, but tiny here, I would suggest we use the same adjective to keep the UI language consistent. I don't have a preference, just would prefer one vs. two unless there is good reason to have to?

If you want to make this fix in a second pass, thats okay too.

Reply:
This is a very good observation: this should be `xsmall`—I absolutely agree.

I absolutely promise I will follow up on this.